sdk/typespec/typespec_client_core/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Changed `ClientOptions::retry` from `Option<RetryOptions>` to `RetryOptions`.
 - Changed `RawResponse::json()` from `async` to synchronous function. The body was already buffered.
 - Changed `RawResponse::xml()` from `async` to synchronous function. The body was already buffered.
+- Moved `AsHeaders`, `FromHeaders`, `Header`, `Headers`, `HeaderName`, and `HeaderValue` to `http::headers` module to align with `typespec_client_core`.
```

Comment:
This is in the `typespec_client_core` changelog - how is moving this aligning with `typespec_client_core`? Should this be `typespec`? Or should this line be in the `azure_core` changelog?